backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/lower/DeepCopyIrTreeWithDescriptors.kt

```diff
@@ -150,7 +150,9 @@ internal class DeepCopyIrTreeWithDescriptors(val targetDescriptor: FunctionDescr
                     /* name                  = */ generateCopyName(oldDescriptor.name),
                     /* kind                  = */ oldDescriptor.kind,
                     /* source                = */ oldDescriptor.source
-            )
+            ).also {
```

Comment:
Is it still necessary? I'm not sure whether it is more correct to copy `isActual` value or not to copy.